server/config/incident.js

```diff
@@ -32,6 +34,7 @@ module.exports = {
       {
         involved: {
           sanitiser: vals => removeEmptyValues('name', vals),
+          fieldType: EXTRACTED,
```

Comment:
Marking these extracted means that they won't be included as part of the json payload and can be intercepted later:

before:
```
{
	"incidentDate": "",
	"name": "",
	"involved": "",
	"evidence": ""
}
`` `
now: 
```{
	"incidentDate": "",
	"involved": "",
	"payload": {
		"name": "",
		"evidence": ""
	}
}
```